app/views/clinicfinders/_search_form.html.erb

```diff
@@ -15,9 +19,18 @@
           <%= f.check_box :medicaid_only, label: 'Show only Medicaid clinics' %>
         </div>
 
-        <div class="col-sm-6 info-form-right">
-          This is an experimental feature.
-          <%#= f.select :gestation, weeks_options, label: "Patient's approx. gestation at appt" %>
+        <div class="col-sm-3 info-form-right">
+          <%= f.select :gestation_weeks,
+                       options_for_select(weeks_options, patient.last_menstrual_period_now_weeks),
+                       label: 'Gestation above',
+                       autocomplete: 'off' %>
+        </div>
+
+        <div class="col-sm-3 info-form-right hide-label">
+          <%= f.select :gestation_days,
```

Comment:
Is thinking in weeks AND days really useful here? Could we get most of the way there if we were just like `weeks.to_i * 7`

Reply:
If it's going to make a large cost difference to our patients, I think it would be useful. Worse case the CMs will only use the week dropdown, best case we catch edge cases where days matter.

Reply:
yeah, I think clinics themselves (at least in DC) tend to work in days, so going with w/d instead of just weeks is the more portable option. Thank you very much for the gut check!